tests/end_to_end/test_slack_integration/data/test_cases/cases_slack_integration.yaml

```diff
@@ -0,0 +1,9 @@
+- name: Slack integration - SSH Brute force attack
```

Comment:
Spaces in the test case name are not recommended.

Reply:
Done in 6c4f812